### PR TITLE
Our values section

### PR DESCRIPTION
**Pull request by matejoslav**

### CODE REVIEW

File: web-app/src/components/FlashCard.js [hunk 1]
```diff
@@ -150,25 +149,24 @@ const FlashCardButton = styled.button`
...
-  height:100%;
+  }*/
+  height: 100%;
 `;
 
 const FlashCardImage = styled.img`
   /*display: flex;
   flex: 1;*/
-  /*width: 100%;*/;
+  /*width: 100%;*/
```
Comment (nicholasmorgan) [inline web-app/src/components/FlashCard.js:165]: I think we should remove commented styling

File: web-app/src/components/aboutPage/OurValuesSection.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import FlashCard from "../FlashCard";
+import LoadingSpinner from "../LoadingSpinner";
+import { getFlashCards } from "../../services/flashCards.service";
+import devices from "../../styles/Devices";
+import { useHistory } from "react-router-dom";
+import NoCardSelectedModal from "../NoCardSelectedModal";
+import { getOurValuesCards } from "../../services/ourValuesCards.service";
+
+const OurValuesSection = (props) => {
+  const [noCardSelectedModalVisible, setNoCardSelectedModalVisible] =
```
Comment (nicholasmorgan) [inline web-app/src/components/aboutPage/OurValuesSection.js:13]: I dont think we need  this variable

File: web-app/src/components/aboutPage/OurValuesSection.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    useState(false);
+
+  const [loading, isLoading] = useState(true);
+
+  const [cardWrapperHeight, setCardWrapperHeight] = useState("100%");
+  const cardWrapperRef = useRef(null);
+
+  let history = useHistory();
+  const [flashCards, setFlashCards] = useState([]);
```
Comment (nicholasmorgan) [inline web-app/src/components/aboutPage/OurValuesSection.js:22]: could they be referred to as value cards to that we don't confuse this with the help section?

File: web-app/src/components/FlashCard.spec.js [hunk 4]
```diff
@@ -42,11 +48,11 @@ describe("the FlashCard component", () => {
...
 
-const renderFlashCard = () => {
+const renderFlashCard = (clickable = true) => {
   const flashCard1 = {
     title: "title 1",
     body: "text 1",
     image: "image url",
   };
-  return render(<FlashCard card={flashCard1} />);
+  return render(<FlashCard clickable={clickable} card={flashCard1} />);
```
Comment (nicholasmorgan) [inline web-app/src/components/FlashCard.spec.js:57]: does this need to be set for the value cards or part of a different ticket?

**Review by nicholasmorgan — CHANGES_REQUESTED** [review]
See comments but looks to be some duplicated code from the flashcards section which I think should be taken out

File: web-app/src/components/FlashCard.spec.js (hunk 4 above)
Comment (matejoslav) [reply web-app/src/components/FlashCard.spec.js:57]: yeah that needs to be part of value cards to disable the + button

**Review by MadelinTom — APPROVED** [review]
looks great

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210813-114249 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210813-114249") to [dev](https://console.bluemix.net/apps/9b529c3e-069e-4b52-b3e4-58f2a13d54b6?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 90df3287ac131b2b15df6baeb1f84a20a4841c39

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210813-160247 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210813-160247") to [test](https://console.bluemix.net/apps/0e5585d7-c9d2-43fc-9810-89b90213879a?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 90df3287ac131b2b15df6baeb1f84a20a4841c39

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210813-162434 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210813-162434") to [prod](https://console.bluemix.net/apps/41d89814-2d1e-4eb9-a923-32ac7859c443?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit 90df3287ac131b2b15df6baeb1f84a20a4841c39

**Pull request merged**